src/_versions.ts

```diff
@@ -11,8 +11,8 @@ export interface TsAppVersion {
 export const versions: TsAppVersion = {
     version: '0.1.0',
     name: 'louis-finesse',
```

Comment:
I would rename this to match the repo name (finesse-frontend)

Reply:
done